modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java

```diff
@@ -752,6 +759,16 @@ void generateApis(List<File> files, List<OperationsMap> allOperations, List<Mode
 
     }
 
+    // checking if apiController file is already existed for spring generator
+    private boolean apiFilePreCheck(String filename, String generator, String templateName, String apiControllerTemplate){
+        File apiFile = new File(filename);
+        if(apiFile.exists() && config.getName().equals(new String(generator)) && templateName.equals(new String(apiControllerTemplate))){
```

Comment:
Why do you use `new String(...)` here? Couldn't you omit these calls?

Reply:
yeah Martin, I will change the same and then raise the PR for original one